spec/functional/win32/security_spec.rb

```diff
@@ -31,14 +34,36 @@
   # support for windows.
   #
   # TODO - Add negative tests once mixlib-shellout has user support
-  it "has_admin_privileges? returns false when running as non-admin" do
-    skip "requires user support in mixlib-shellout"
+  describe "running as non admin user" do
+    let(:user) { "security_user" }
+    let(:password) { "security" }
+    before do
+      include Chef::Mixin::WideString
```

Comment:
we should definitely add a comment that this needs to be replaced when this code ends up in a mixin and refers to #5832.